front-end/src/api/authApi.ts

```diff
@@ -28,4 +30,26 @@ export class AuthApi {
     return (await nonAuthAppApi.post("/users/oAuthLogin", oAuthLoginDto))
       .data as ApiResponseDto<UserTokensDto>;
   }
+
+  public static async verifyTwoFactor(verifyTwoFactorDto: VerifyTwoFactorDto) {
+    return (
+      await nonAuthAppApi.post("/auth/verifyTwoFactor", verifyTwoFactorDto)
+    ).data as ApiResponseDto<UserTokensDto>;
+  }
+
+  public static async enableTwoFactor() {
+    const token = localStorage.getItem("token"); // Adjust this line as necessary
```

Comment:
don't need to do this. do appApi.post(); Don't need to configure header .

Reply:
> don't need to do this. do appApi.post(); Don't need to configure header .

revised
ready to merge